lib/screens/home/widgets/grid_item.dart

```diff
@@ -0,0 +1,28 @@
+import 'package:flutter/material.dart';
```

Comment:
My suggestion for how the grid items could look like is some form of a relevant picture as a background that's been tinted with the appropriate color. Could just use placeholder images for now. 

This is of course a suggestion for the future, not important.